docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,305 @@
+    5. `role` (OPTIONAL, string). Role of the contact person. This field should only be set when the `name` field
+    is set.
+
+#### 2. Models
+
+1. `models` (OPTIONAL, list[ModelCard]). A list of model cards (as defined below) or `!include`s of a yaml
+file containing a model card. This model card can for example be a model card described in the next section
+or a model card from Hugging Face. There can be multiple model cards, meaning multiple models are used.
+
+#### 3. Assessments
+
+There can be multiple assessments. For each assessment the following fields are present:
+
+1. `name` (REQUIRED). The name of the assessment.
+2. `date` (REQUIRED). The date at which the assessment is completed.
+3. `contents`. There can be multiple items in contents. For each item the following fields are present:
+
+    1. `question` (REQUIRED). A question.
+    2. `answer` (REQUIRED). An answer.
+    3. `remarks` (OPTIONAL). A field to put relevant discussion remarks in.
+    4. `authors`. There can be multiple names. For each name the following field is present.
+        1. `name` (OPTIONAL). The name of the author of the question.
+    5. `timestamp` (OPTIONAL). A timestamp of the date and time of the answer.
+
+### `model_card`
+
+A `model_card` contains the following information.
+
+1. `algorithm-registers` (OPTIONAL, list[string]). If this algorithm is registered in an algorithm register,
+this field should contain a URI to the corresponding record in the register. There can be multiple registers.
+A URI could look like `https://huggingface.co/org/modelid` or `https://algoritmes.overheid.nl/nl/algoritme/modelid`.
+2. `language` (OPTIONAL, list[string]). If relevant, the natural languages the model supports in [ISO 639](https://www.iso.org/iso-639-language-code).
+    There can be multiple languages.
+3. `license`(REQUIRED, string). Any license from the [open source license list](https://opensource.org/license)
+[^1]. If the license is NOT present in the license list this field must be set to 'other' and the following
+two fields will be REQUIRED.
+
+    1. `license_name` (string). An id for the license.
+    2. `license_link` (string). A link to a file of that name inside the repo, or a URL to a remote file containing the license
+    contents.
+
+4. `library_name` (OPTIONAL, string). Any library from the [library list](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts).
+    If the library is NOT present in the library list, the following fields CAN be set.
+    1. `library` (OPTIONAL, string). If a library is used that is not in the above list, you can provide
+    the name of the library used here.
+    2. `library_link` (OPTIONAL, string). If a libary is used that is not in the above list, you can
+    provide a URI to the repo of the library used here.
+5. `tags` (OPTIONAL, list[string]). Tags with keywords to describe the project. There can be multiple tags.
+
+### 2. `model_index`
```

Comment:
```suggestion
#### `model_index`
```
I think this is a typo, should be one level under model (maybe even on the same level as `tags` but i get you want to have special highlight for this and therefore to make it big as it can be source of discussion